doc/doxygen/Makefile

```diff
@@ -147,20 +146,10 @@ clean-tooling:
 todo:
 	@grep -Eor "@needsref [a-zA-Z._-]+" ../ | sort | uniq | sed 's|..//||' | column -s ':' -t
 
-##
-## File targets
-##
-
-src/styles.css: src/styles.scss $(SASS)
-	./node_modules/.bin/sass --no-source-map ./src/styles.scss ./src/styles.css
-
 #
 # Tooling
 #
 
-$(SASS):
-	npm ci
```

Comment:
[fix] Since this is gone we can also remove the `rm -rf node_modules` from the `clean-tooling` target.

Reply:
Addressed 2723f2dbb4f69b92e0c67d62d4365d6302917e9d